README.md

```diff
@@ -218,6 +218,144 @@ OpenStack | N | N | Y | N | N
+   ```
+   ./sts-preflight create --infra-name <aws_infra_name> --region <aws_region> --credentials-requests-to-roles /path/to/credreqs.yaml/downloaded/in/step/2   
+   ```
+8. Currently, the [sts-preflight](https://github.com/sjenning/sts-preflight) tool does not put condition on the created IAM role to allow only specific namespace/ServiceAccountName from assuming the role. We need to manually modify each role to achieve this. Please note that role name might be truncated if it is more than 64 characters
+
+    i. Modify trust relationship of the role <aws_infra_name>-openshift-image-registry-installer-cloud-credentials to have the following condition. 
+        
+    ```json
+    "Condition": {
+      "StringEquals": {
+        "s3.us-east-1.amazonaws.com/<aws_infra_name>-installer:sub": [
+          "system:serviceaccount:openshift-image-registry:registry",
+          "system:serviceaccount:openshift-image-registry:cluster-image-registry-operator",
+        ]
+      }
+    }         
+    ```
+   
+    ii. Modify trust relationship of the role <aws_infra_name>-openshift-ingress-operator-cloud-credentials to have the following condition.
+
+    ```json
+    "Condition": {
+      "StringEquals": {
+        "s3.us-east-1.amazonaws.com/<aws_infra_name>-installer:sub": [
+          "system:serviceaccount:openshift-ingress-operator:ingress-operator",
+        ]
+      }
+    }         
+    ```
+   iii. Modify trust relationship of the role <aws_infra_name>-openshift-cluster-csi-drivers-ebs-cloud-credentials to have the following condition.
+
+    ```json
+    "Condition": {
+      "StringEquals": {
+        "s3.us-east-1.amazonaws.com/<aws_infra_name>-installer:sub": [
+          "system:serviceaccount:openshift-cluster-csi-drivers:aws-ebs-csi-driver-operator",
+          "system:serviceaccount:openshift-cluster-csi-drivers:aws-ebs-csi-driver-controller-sa",          
+        ]
+      }
+    }         
+    ```
+   iv. Modify trust relationship of the role <aws_infra_name>-openshift-machine-api-aws-cloud-credentials to have the following condition.
+
+    ```json
+    "Condition": {
+      "StringEquals": {
+        "s3.us-east-1.amazonaws.com/<aws_infra_name>-installer:sub": [
+          "system:serviceaccount:openshift-machine-api:machine-api-controllers",          
+        ]
+      }
+    }         
+    ```
+   
+9. Copy the manifests created in the step 7 and put them in the same location as install-config.yaml in the `manifests` directory
+   ```
+   cp _output/manifests/* ./path/to/dir/with/install-config.yaml/manifests/
+   ```
+10. Copy the private key for the ServiceAccount signer and put it in the same location as install-config.yaml
```

Comment:
for this one i think we need to create the `tls` directory IIRC. so maybe the step should be `cp -a`